### ISSUE TITLE
TypeError since Java is missing in 0.26

### ISSUE THREAD

**Issue by sino1641** [redhat-developer/vscode-xml#954]
VSC about:
```
Version: 1.85.0-insider (user setup)
Commit: eb56719ae7653bff54989385818ef39e1cd73d29
Date: 2023-11-24T15:21:39.004Z
Electron: 25.9.4
ElectronBuildId: 25127168
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
```

Extension: v0.26.2023111808 pre-release
When I try to open any .xml file, there is a 
`TypeError: Cannot read properties of null (reading '1'). Cannot start XML language server, since Java is missing.` 

The extension already downloaded `lemminx-win32.exe` under extension folder.
I also tried to turn on `"xml.server.preferBinary": true` and reloaded VSC but error still occurs.
Plz advice.

**Comment by sino1641** [comment]
Happy new year!
Is there any update?

**Comment by angelozerr** [comment]
> Happy new year!

Thanks, Happy new year too!

> Is there any update?

I cannot reproduce your issue. It seems that there is a variable which is null in https://github.com/redhat-developer/vscode-xml/blob/184bdd95a61e82612416141c92b29c18dcdc1427/src/server/binary/binaryServerStarter.ts#L31 but it is hard to find out.

Have you the same problem with old version of vscode-xml?

**Comment by sino1641** [comment]
Hi @angelozerr , Much thanks for help!
It seems fixed after I upgrading to vscode 1.86.0
